src/Generator/ActionParameterGenerator.php

```diff
@@ -42,8 +43,15 @@ public function generate(): void
                 $parameters = [];
 
                 if (!empty($method['requestBody']['content'])) {
-                    $requestModel          = $method['requestBody']['content']['application/json']['schema']['$ref'];
-                    $requestModelClassName = empty($requestModel) ? null : basename($requestModel);
+                    $schema = $method['requestBody']['content']['application/json']['schema'];
```

Comment:
Wont this fail if we don't have application/json available? Maybe check for that too

Reply:
Yeah, currently we handle only json requests. And trying to postpone the implementation of other types as much as possible